environment/spin-kube.sh

```diff
@@ -32,6 +32,28 @@ install_cert_manager() {
   kubectl wait --for=condition=available --timeout=${READINESS_TIMEOUT} deployment/cert-manager-webhook -n cert-manager
 }
 
+install_kube_prometheus() {
```

Comment:
As a follow-up, are you thinking we might update `spin-kube-k8s.sh` to toggle installation of prom/grafana if `OUTPUT=prometheus`?

Reply:
That makes sense. I can add that!